### DISCUSSION TITLE
sharing reference files

### DISCUSSION THREAD

**Discussion by stef-mueller** [dnanexus-rnd/liftover_plink_beds#1] [Q&A]
Hello,

would it please be possible that you share the reference files (ie. .chain file and .fa.gz with .dict and .fai) which were used to run this workflow on UKBB genotypes in DNAnexus environement? Trying to replicate the workflow, I am encountering errors due to unmatched reference contigs - specifically extraction of PAR1 and PAR2 in task "merge_and_split_by_chr". I would be very thankful indeed.

Many thanks, Stefanie

**Comment by pjgreer** [comment]
I hope you figured this out before now, but the issue you are having is likely due to chromosome naming mismatch in the chainfile. see my other discussion post here.
